### PR TITLE
fix(argo-cd): helm3 install does not have flag --name

### PR DESCRIPTION
**Pull request by cyril-corbon**
Details:
just fix typo in the readme in the  `## Installing the Chart` part

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#versioning)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [x] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

**Review by mkilchhofer — DISMISSED** [review]
Although this is only a minimal change, we need a version bump to satisfy CI. So please update the Chart.yaml and bump the patch version (and include the changelog annotation).
Also please add this to the README.md.gotpl as the README.md is autogenerated by [norwoodj/helm-docs](https://github.com/norwoodj/helm-docs/)

**Comment by cyril-corbon** [comment]
@mkilchhofer I just applied the modification. thanks for the review :smile:

File: charts/argo-cd/Chart.yaml [hunk 1]
```diff
@@ -21,4 +21,4 @@ dependencies:
     condition: redis-ha.enabled
 annotations:
   artifacthub.io/changes: |
-    - "[Changed]: Update to app version 2.1.6"
+    - "[Changed]: Update to app version 2.1.7"
```
Comment (mbevc1) [inline charts/argo-cd/Chart.yaml:24]:
```suggestion
    - "[Changed]: README update to reflect correct helm install syntax"
```

**Comment by mbevc1** [comment]
Thanks for the contribution :+1: . Looks good, could you please sign accepted commit?

**Comment by cyril-corbon** [comment]
Done :wink:

File: charts/argo-cd/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: v2.1.6
+appVersion: v2.1.7
```
Comment (mbevc1) [inline charts/argo-cd/Chart.yaml:2]: We shouldn't be bumping ArgoCD image version as latest is still `2.1.6`, only chart version.
```suggestion
appVersion: v2.1.6
```

**Review by mbevc1 — CHANGES_REQUESTED** [review]
This change is bumping to invalid ArgoCD version

**Review by mbevc1 — APPROVED** [review]
Thanks :+1:

**Pull request merged**